src/components/Tab/Tab.module.css

```diff
@@ -0,0 +1,69 @@
+@import "../../styles/globals.css";
+@import "../../styles/variables.css";
+
+.Tab {
+  color: white;
+  height: 41px;
+  min-width: 115px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 0px;
+  border-bottom-left-radius: 0px;
+  border-width: 1.3px;
+  font-size: 16px;
+  font-family: "Raleway", Arial, Helvetica, sans-serif;
+  font-weight: 600;
+}
+
+.Tab:focus {
+  outline: none;
+}
+
+.Tab:active {
+  box-shadow: none;
+  transform: scale(0.99);
+}
+
+.primary {
+  background-image: linear-gradient(
+    -180deg,
+    var(--color-light-gray-2),
+    var(--color-light-gray-3) 90%
+  );
+  box-shadow: 0 1px 15px rgba(0, 0, 0, 0.08);
+}
+
+.primary:active {
+  border-top: 1.3px solid var(--color-gray-2);
+  border-left: 1.3px solid var(--color-gray-2);
+}
+
+.disabled {
+  color: #888;
+  background-image: linear-gradient(
+    -180deg,
+    var(--color-light-gray-2),
+    var(--color-light-gray-3) 90%
+  );
+  box-shadow: 0 1px 15px rgba(0, 0, 0, 0.08);
+}
+.samfundet {
+  background-color: #ba202c;
```

Comment:
bruk variablene fra fila. blir mer ryddig! Da kan vi endre de globale fargene for uka f.eks bare ett sted, i stedefor å måtte endre fargen alle steder vi har brukt de.

Reply:
samme på uka, isfit og annet